src/Web/Controllers/AccountController.cs

```diff
@@ -81,27 +97,73 @@ public async Task<IActionResult> SignIn(LoginViewModel model, string returnUrl =
             }
             ViewData["ReturnUrl"] = returnUrl;
 
+            return await SignInUser(model, returnUrl, false);
+        }
+
+        private async Task<IActionResult> SignInUser(LoginViewModel model, string returnUrl, bool useAP)
+        {
             var applicationUser = new ApplicationUser
             {
                 UserName = model.Email
             };
+            string hashedPassword = _userManager.PasswordHasher.HashPassword(applicationUser, model.Password);
 
-            var signIn = new SignIn
+            if (useAP)
             {
-                SignInId = Guid.NewGuid().ToString(),
-                PasswordHash = _userManager.PasswordHasher.HashPassword(applicationUser, model.Password),
-                MerchantLocalDate = DateTimeOffset.Now,
-                CustomerLocalDate = model.DeviceFingerPrinting.ClientDate,
-                UserId = model.Email,
-                DeviceContextId = model.DeviceFingerPrinting.SessionId,
-                AssessmentType = AssessmentType.Protect.ToString(),
-                CurrentIpAddress = _contextAccessor.HttpContext.Connection.RemoteIpAddress.ToString()
-            };
+                var user = new AccountProtection.User()
+                {
+                    UserType = "Consumer",
+                    Username = model.Email,
+                    PasswordHash = hashedPassword
+                };
 
-            var signInResponse = await _fraudProtectionService.PostSignIn(signIn);
+                var device = new AccountProtection.DeviceContext()
+                {
+                    sessionID = model.DeviceFingerPrinting.SessionId,
+                    IpAddress = _contextAccessor.HttpContext.Connection.RemoteIpAddress.ToString()
+                };
 
-            var fraudProtectionIO = new FraudProtectionIOModel(signIn, signInResponse, "SignIn");
-            TempData.Put(FraudProtectionIOModel.TempDataKey, fraudProtectionIO);
+                var metadata = new AccountProtection.EventMetadataAccountLogin()
+                {
+                    Name = "AP.AccountLogin.Metadata",
```

Comment:
Why is this Name needed?  Isn't it enough that the SignIn object has a name+version?  I'm asking, I don't actually know either way.

Reply:
It was originally called out as a required field in the documentation, but this seems to have changed as I can omit it and calls still succeed.  I've removed it.